sycl/doc/extensions/experimental/sycl_ext_intel_fp_control.asciidoc

```diff
@@ -0,0 +1,176 @@
+= sycl_ext_intel_fp_control
```

Comment:
Why do we want to put this extension in this location?  All the other features that are specific to ESIMD are documented in "sycl/doc/extensions/supported/sycl_ext_intel_esimd".

Reply:
It was proposed to have this extension for generic SYCL. Even though currently gpu driver supports these modes only for vector backend, it is possible to add that support for scalar backend as well.

Reply:
Do we have concrete plans to add support to the scalar backend, or is this just something that we might theoretically do?

If there are no concrete plans, I'd rather move this spec to the esimd folder.  I think there is a danger that people will see the spec and then assume this functionality works for regular SYCL kernels.  I know there is a disclaimer in the "Backend support status" section, but it's easy to miss.

Reply:
@gmlueck Sorry, I missed your comment. Yes, there are concrete plans to support that for scalar backend.